lib/src/proto/livekit_rtc.pbjson.dart

```diff
@@ -213,6 +213,15 @@ const SignalResponse$json = const {
       '9': 0,
       '10': 'roomUpdate'
     },
+    const {
```

Comment:
Wow, a bunch of magic numbers and strings. What are these?

Reply:
these are generated protobuf files :)